### PR TITLE
Invite Dialog

### PR DESCRIPTION
**Pull request by asutula**

### CODE REVIEW

File: App/Containers/ThreadInvite.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ class ThreadInvite extends React.PureComponent {
   }
 
   cancel = () => {
+    // TODO: remove thread takes thread name... what is key?
     this.props.removeThreadRequest(this.state.key)
```
Comment (asutula) [inline App/Containers/ThreadInvite.js]: @andrewxhill this function to remove the thread needs to take the thread name (until we fix that API)... wasn't sure what `this.state.key` represents, but let's make sure we're passing thread name here?

Comment (andrewxhill) [reply App/Containers/ThreadInvite.js]: i think we'll have a fix for it later today from the go side, so maybe just let it be until framework update

**Comment by asutula** [comment]
Ok, besides the question i left for @andrewxhill, this is good to go.

File: App/Containers/ThreadInvite.js (hunk 1 above)
Comment (asutula) [reply App/Containers/ThreadInvite.js]: cool

**Comment by andrewxhill** [comment]
Let's do it! Team Thread in 3. 2. 1.... CircleCI com'on! 3.. 2..

File: App/Sagas/TextileSagas.ts [hunk 2]
```diff
@@ -389,6 +389,11 @@ export function * addExternalInvite (action: ActionType<typeof ThreadsActions.ad
   }
 }
 
+export function * presentShareInterface(action: ActionType<typeof ThreadsActions.addExternalInviteSuccess>) {
+  const { link } = action.payload
+  yield call(Share.share, { title: 'Join my thread on Textile!', message: link })
+}
```
Comment (andrewxhill) [inline App/Sagas/TextileSagas.ts]: link should be appended with "&from=some-screen-name" before this point.

**Pull request merged**